internal/services/media/media_transform_resource.go

```diff
@@ -82,6 +84,7 @@ func resourceMediaTransform() *pluginsdk.Resource {
 						"on_error_action": {
 							Type:         pluginsdk.TypeString,
 							Optional:     true,
+							Default:      string(encodings.OnErrorTypeStopProcessingJob),
```

Comment:
was this the api default?

Reply:
Yes, this is tested at https://github.com/hashicorp/terraform-provider-azurerm/pull/21279/files#diff-4967a1e50d98c183ca4362e83543c02b2e3aec1e2b10f2ba2bcb0a0291800e13R195-R206